libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -1266,18 +1270,30 @@ void CompilerUtils::rightShiftNumberOnStack(unsigned _bits)
 
 unsigned CompilerUtils::prepareMemoryStore(Type const& _type, bool _padToWords)
 {
+	solAssert(
+		_type.sizeOnStack() == 1,
+		"Memory store of types with stack size != 1 not implemented (Type: " + _type.toString(true) + ")."
```

Comment:
Yes exactly, this should be reworded to "not allowed" or something else. I mean "not implemented" is not a wrong statement, but it would suggest that it should be implemented, which is not the case :)